### DISCUSSION TITLE
Rupture of shell elements

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#1518] [Q&A] [answered]
Hi, i'm trying to perform a crash analysis with a model with few elements but if i don't put a limit on the maximum timestep it delete the shell elements. Are there any mandatory keywords to insert to perform a crash analysis? Or it could be a problem of the element section?

**Comment by MarianBulla** [comment, marked as answer]
Hi,
it seems to be a problem of element section or material definition.
Usually, you don't need to define a maximum timestep. In that case the solver uses the one, calculated by the starter (you can check it in the _0000.out file).

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by Schwoertzig** [comment]
Hi @FGarelli , you may also increase frequency output to post-treat stress/deformation and verify the expected behavior. It can help to check your material model. Unit translation (if defined) for material parameters, or load case,  could also be checked in Starter listing file (*_0000.out) 

- H3D frequency output (animation) : see /H3D/DT option in Engine file *_0001.rad ; and /H3D/SHELL/* options to define what to output.
- TH frequency output (time histories) : see /TFILE option in Engine file *_0001.rad.  (see also /TH/SHELL starter option to define TH)

Regards
Thierry
